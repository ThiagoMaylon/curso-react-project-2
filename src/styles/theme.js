export const theme = {
   color: {
        mainBG: 'red',
        secondaryBG: 'blue'
   },
   fonts: {}
}