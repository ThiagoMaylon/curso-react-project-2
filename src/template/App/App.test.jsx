import { screen } from '@testing-library/react';
import Home from '.';
import {renderTheme} from '../../styles/render-theme'


test('renders learn react link', () => {
  const {debug} = renderTheme(<Home />);
  const headingContainer = screen.getByRole('heading', {name: 'ol√° mundo'}).parentElement;

  expect(headingContainer).toHaveStyleRule({
    background: 'red',
  });
  
  expect(headingContainer).toMatchSnapshot();

});
