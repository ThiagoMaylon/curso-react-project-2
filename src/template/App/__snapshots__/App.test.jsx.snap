// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  class="sc-bcXHqe TLTrD"
>
  <h1>
    ol√° mundo
  </h1>
</div>
`;
